### DISCUSSION TITLE
Welcome to wargabantuwarga.com Discussions!

### DISCUSSION THREAD

**Discussion by zainfathoni** [kawalcovid19/wargabantuwarga.com#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ariabrata** [comment]
Wargabantuwarga. Comini ide bagus, memberikan informasi seputar hal yg berkaitan dg covid 19 ini, mungkin akan lebih baik lagi wargabantuwarga. Com ini dapat menjadi media pembantu untuk pencegahan penyebaran covid 19 lebih luas lagi... Antar warga bisa menjadi terkoneksi satu sama lain, menjadi pengingat antar warga.... Mungkin dengan ini bisa menekan laju penyebaran covid 19 ini, agar kita semua dapat melanjutkan hidup, ekonomi berjalan dengan baik, nyawa manusia menjadi lebih berharga... Terima kasih

**Comment by ramaadtym** [comment]
@zainfathoni saya punya ide untuk desain website nya seperti ini mas. mungkin bisa membantu, :) 

https://www.figma.com/file/I5xwQlR6adtwiIhgDrVPk0/Warga-Bantu-Warga?node-id=1%3A13

**Reply by naddd03** [reply]
Keren mass. Izin kasih saran mas, mungkin untuk ukuran text pada pilihan layanan bisa diperbesar dan icon pada floating menu diberi label, mengingat yang akses wargabantuwarga mencakup semua kalangan termasuk orangtua hehe biar lebih mudah dipahamii

**Reply by ramaadtym** [reply]
oke noted @naddd03 :) terima kasih masukkan nya

**Reply by baraeb92** [reply]
Mas @ramaadtym di epic ini kita mentioned beberapa issue sama site design yang skrng, mainly karena dari analytic kita, kita liat kebanyakan user itu menggunakan mobile browser untuk mengakses informasi ini: https://github.com/kawalcovid19/wargabantuwarga.com/issues/68

Jadi nanti kita akan migrasikan data data yang skrng masih menggunakan Google Sheet untuk diganti ke tampilan yang kita design sendiri, agar lebih mobile friendly. 

Jadi minta tolong dicheck juga ya.

**Reply by baraeb92** [reply]
Kalau boleh minta tolong kita ngedesignnya pake mobile first design conceptnya dulu, jadi dibikin versi mobilenya baru terus bikin versi desktop. 

Dan juga untuk design dengan touch interface, jadi ukuran button dan interactive componentnya cukup besar untuk pengguna yang pake jari untuk press dan navigasi, karena g bisa sepresisi pake mouse untuk clickingnya.

**Reply by ramaadtym** [reply]
hmm, oke noted2 @baraeb92 . kalo boleh tau stack yang dipakai apa ya? React or Vue? terus untuk responsive nya biasanya pake apa? bootstrap atau bulma atau ada yang lain?

**Reply by baraeb92** [reply]
Kalau untuk tech stacknya mungkin mas @zainfathoni bisa bantu jawab. Sebagian bisa dicheck disini: https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/package.json

**Reply by zainfathoni** [reply]
Makasih banyak Mas @ramaadtym.

Iya, kami sedang dalam tahap implementasi desain yg baru. Saya sendiri sekarang sedang mengerjakan #75 dengan desain yang saya comot dari [Tailwind UI](https://tailwindui.com) karena kebetulan saya pribadi sudah punya lisensinya.

Terkait teknologi yang digunakan, kami pakai Tailwind CSS Mas, jadi kalau bisa design system-nya yang sejalan dengan [prinsip-prinsip Tailwind CSS seperti mobile-first, dsb.](https://tailwindcss.com/docs/responsive-design)

Berikut ini [Resources](https://tailwindcss.com/resources)-nya.

**Reply by zainfathoni** [reply]
Terkait _technology stack_ yang dipakai, untuk memperjelas penggunaan [beberapa library ini](https://github.com/kawalcovid19/wargabantuwarga.com/blob/main/package.json), kami menggunakan [Next.js](https://nextjs.org) yang dibangun di atas [ReactJS](https://reactjs.org), dengan styling menggunakan [Tailwind CSS](https://tailwindcss.com) yang dihubungkan dengan React menggunakan [Headless UI](https://headlessui.dev), serta [Hero Icons](https://heroicons.com) dan [Hero Patterns](https://www.heropatterns.com) sebagai aset-aset dasarnya.

Untuk meningkatkan performa web, menurut pendapat saya, kita harus menghindari gambar dengan ukuran terlalu besar, oleh karena itu, sebisa mungkin untuk Hero image kita gantikan dengan [Hero Patterns](https://www.heropatterns.com) ini.

**Reply by zainfathoni** [reply]
Sebagai contoh, website [KawalCOVID19](https://kawalcovid19.id) ini tidak menggunakan gambar tambahan selain logo, itu juga salah satu alasannya.

**Reply by ramaadtym** [reply]
Oke saya coba pelajari asset nya ya mas @zainfathoni . mungkin sebagai alternatif, saya ambil icon-icon nya via https://iconify.design/ sudah support react juga

**Reply by ramaadtym** [reply]
https://www.figma.com/file/I5xwQlR6adtwiIhgDrVPk0/Warga-Bantu-Warga?node-id=25%3A190 ini update dari saya mungkin bisa dicek dulu untuk yang mobile design nya cc: @baraeb92 @naddd03 @zainfathoni

**Reply by mazipan** [reply]
Ini cukup bagus struktur informasinya.
Cuma bisa apa gak nya dibuat dengan struktur data sekarang, mas @zainfathoni mungkin bisa cek2 dulu.

Kalau oke, dibuat issue terpisah aja per masing-masing screen, buat chunking kerjaannya.

**Reply by zainfathoni** [reply]
Iya Mas, ini sedang dieksplorasi dulu.
Untuk saat ini, saya fokus ke menuntaskan #31 dulu. 😁

**Reply by mazipan** [reply]
Iya, better dikelarin dulu untuk prepare struktur data yang lebih oke buat di consume.

Abis itu kita bisa define UI nya dengan lebih baik.

**Reply by zainfathoni** [reply]
Sementara itu, Mas @baraeb92 juga sedang kerja bareng tim UI/UX designer dari Kitabisa untuk mematangkan ini juga. 😁
